src/main/scala/machines/Internal.scala

```diff
@@ -3,4 +3,31 @@ package machines
 import regex._
 import dfa._
 
-// TODO: Add your code below
+// Part 1
+given Conversion[Char, RegularLanguage] = n => Character(n)
```

Comment:
I did this in a slightly different way, instead of writing n => Character(n) i just wrote Character(_)